### DISCUSSION TITLE
菜单栏没有归档、分类、标签等界面

### DISCUSSION THREAD

**Discussion by leeima** [LIlGG/halo-theme-sakura#425] [General]
### 目前使用的 Sakura 版本？

 2.0.0-beta.2

### 目前使用的 halo 版本？

2.8

### 建议/问题

![image](https://github.com/LIlGG/halo-theme-sakura/assets/84554107/ee9300a2-3564-4c46-9688-eb590353021d)
菜单栏没有这些菜单页面，无法编辑添加
![image](https://github.com/LIlGG/halo-theme-sakura/assets/84554107/fc981357-01e1-4a18-8aeb-bee19ab713de)

**Comment by LIlGG** [comment]
主题并不提供菜单，菜单需要自己去新建。

**Comment by LIlGG** [comment]
如下所示
<img width="1920" alt="image" src="https://github.com/LIlGG/halo-theme-sakura/assets/31335418/484f60e5-323c-42b4-a1c0-0981772a17f2">
